libs/external-db-mysql/src/index.ts

```diff
@@ -9,20 +9,22 @@ export * as testResources from '../tests/e2e-testkit/mysql_resources'
 import { MySqlConfigValidator } from './mysql_config_validator'
 import { DatabaseFactoryResponse, DbConnector } from '@wix-velo/velo-external-db-commons'
 import init from './connection_provider'
+import { Logger } from '@wix-velo/external-db-logger'
 
 export const driver = () => require('../tests/drivers/sql_filter_transformer_test_support')
 export const opsDriver = () => require('../tests/drivers/db_operations_test_support')
 
 
 export class MySqlConnector extends DbConnector {
-    constructor() {
+    constructor(logger: Logger) {
         super(MySqlConfigValidator, init)
         this.type = 'mysql'
+        this.logger = logger
     }
 }
 
-export const mySqlFactory = async(config: any, options: any): Promise<DatabaseFactoryResponse> => {
-    const connector = new MySqlConnector()
-    const { connection, cleanup, ...providers } = await connector.initialize(config, options)
+export const mySqlFactory = async(config: any, logger: Logger, options: any): Promise<DatabaseFactoryResponse> => {
```

Comment:
Using the same logger instance for all prints is a common practice. 
And it will allow us to define the settings of the prints in only one place instead of all console.log appearances.